README.md

```diff
@@ -16,15 +16,73 @@
 # limitations under the License.
 #
 -->
+
 # Apache APISIX Dashboard
 
-Dashboard for [Apache APISIX](https://github.com/apache/incubator-apisix-dashboard)
+Dashboard for [Apache APISIX](https://github.com/apache/apisix-dashboard)
 
-## Deploy with Docker (currently)
+## Deploy with Docker
 
 Please refer to [Deploy with Docker README](./compose/README.md)
 
-## More
+## Deploy Manually
+
+### Build the manager-api
+
+The `manager-api` is used to provide APIs for Dashboard, just like a bridge between the Apache APISIX and the Dashboard. Here are the steps to build it manually:
+
+1. We need `MySQL/Golang` to be preinstalled.
+
+```sh
+# e.g Initialization for MySQL
+$ mysql –uroot –p123456 
+> source $PROJECT_ROOT_FOLDER/api/script/db/schema.sql
```

Comment:
how about remove `$PROJECT_ROOT_FOLDER`? And `apisix-dashboard` is good for me.

Reply:
OK